src/worker.h

```diff
@@ -476,6 +476,8 @@ class NetworkThread : public WorkerThread {
   // Calculate worker thread specific bandwidth.
   virtual float GetDeviceCopiedData() { return GetCopiedData() * 2; }
 
+  string GetThreadTypeName() { return "Network IO Thread"; }
```

Comment:
Are these Getters used anywhere?

Reply:
Yea these are used in the AddLog, AddProcessError, and AddDiagnosis functions in the parent class in order to append the thread name and id to the start of each artifact message.